edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/TransportDocumentTO.java

```diff
@@ -204,6 +204,13 @@ private void validateNegotiableBL() {
     Predicate<DocumentPartyTO> isEndorseeParty = p -> p.partyFunction().equals(PartyFunction.END.name());
     var endorseeParties = nullSafeStream(documentParties).filter(isEndorseeParty).toList();
     verifyConsistency(endorseeParties.size() < 2, "Negotiable B/Ls cannot more than one endorsee party");
+
+    Predicate<DocumentPartyTO> isConsigneeOrCFF = p -> p.partyFunction().equals(PartyFunction.CN.name())
```

Comment:
I know that Consignee = CN partyFunctionCode - but why the CFF in the name? If you are using abbreviations - why not use DDS (as this is the PartyFunctionCode)?

Reply:
Because `CFF` is a more natural abbreviation for `Consignee Freight Forwarder`. Though I can it to DDF if that matters to you